src/Sylius/Behat/Context/Ui/Admin/ManagingCustomersContext.php

```diff
@@ -0,0 +1,132 @@
+use Sylius\Behat\Page\Admin\Customer\CreatePageInterface;
+use Sylius\Behat\Service\NotificationCheckerInterface;
+use Sylius\Component\User\Model\CustomerInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+final class ManagingCustomersContext implements Context
+{
+    const RESOURCE_NAME = 'customer';
+
+    /**
+     * @var IndexPageInterface
+     */
+    private $indexPage;
+
+    /**
+     * @var CreatePageInterface
+     */
+    private $createPage;
+
+    /**
+     * @var NotificationCheckerInterface
+     */
+    private $notificationChecker;
+
+    /**
+     * @var UpdatePageInterface
+     */
+    private $updatePage;
+
+    /**
+     * @param CreatePageInterface $createPage
+     * @param IndexPageInterface $indexPage
+     * @param UpdatePageInterface $updatePage
+     * @param NotificationCheckerInterface $notificationChecker
+     */
+    public function __construct(
+        CreatePageInterface $createPage,
+        IndexPageInterface $indexPage,
+        UpdatePageInterface $updatePage,
+        NotificationCheckerInterface $notificationChecker
+    ) {
+        $this->createPage = $createPage;
+        $this->indexPage = $indexPage;
+        $this->updatePage = $updatePage;
+        $this->notificationChecker = $notificationChecker;
+    }
+
+    /**
+     * @Given I want to create a new customer
+     */
+    public function iWantToCreateANewCustomer()
+    {
+        $this->createPage->open();
+    }
+
+    /**
+     * @When /^I specify its ([^"]*) as "([^"]*)"$/
+     */
+    public function iSpecifyItsFirstNameAs($elementName, $value)
+    {
+        $this->createPage->specifyElementValue($elementName, $value);
+    }
+
+    /**
+     * @When I add it
+     */
+    public function iAdd()
+    {
+        $this->createPage->create();
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully created
+     */
+    public function iShouldBeNotifiedItHasBeenSuccessfulCreation()
+    {
+        $this->notificationChecker->checkCreationNotification(self::RESOURCE_NAME);
+    }
+
+    /**
+     * @Given the customer :customer should appear in the store
+     */
+    public function thisCustomerShouldAppearInTheStore(CustomerInterface $customer)
+    {
+        $this->updatePage->open(['id' => $customer->getId()]);
+
+        Assert::true(
+            $this->updatePage->assertElementHasValue('email', $customer->getEmail()),
+            sprintf('Customer with email %s should exist but it does not.', $customer->getEmail())
+        );
+    }
+
+    /**
+     * @When I select :value from gender
+     */
+    public function iSelectGender($value)
+    {
+        $this->createPage->chooseGender($value);
+    }
+
+    /**
+     * @Given the customer :customer with this birthday and this gender should appear in the store
+     */
+    public function theCustomerWithThisBirthdayAndThisGenderShouldAppearInTheStore(CustomerInterface $customer)
```

Comment:
In fact we don't even check, whether birthday and gender was correctly set, as `$customer` will be loaded from database, based on changes we've made. 

Let's say customer is a man. We change it's gender to "Female", but there's some bug and it isn't saved into database. Then this step will receive the customer, whose gender is still "Male".